spaces/S000136/properties/P000019.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000136
+property: P000019
+value: false
+refs:
+- doi: 10.1007/978-1-4612-6290-9_6
+  name: Counterexamples in Topology
+---
+
+{S000137} is a closed subspace of $X$, but {S000137} is not {P000019}
```

Comment:
Even though it works either way, we like to use the shorter versions in the internal links, i.e., "{S137}" and "{P19}".  Makes it easier to work with when editing things.